src/main/java/com/github/takayoshi24/cinema/seance/SeanceDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.github.takayoshi24.cinema.seance;
+
+import com.github.takayoshi24.cinema.movie.Movie;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+public record SeanceDTO(UUID id, Movie movie, ZonedDateTime startingAt, Integer room){
```

Comment:
Don't use internal classes (like Movie here) in DTOs. Instead, MovieDTO should be used here.